remo/voting/tasks.py

```diff
@@ -138,7 +141,7 @@ def create_rotm_poll():
                                     end__range=[start, end])
 
     if not now().date() > ROTM_NOMINATION_END_DATE or rotm_poll.exists():
```

Comment:
I changed the name of the flag to avoid confusion with the name of the waffle switch.